### ISSUE TITLE
bootctl status ignores --json flag

### ISSUE THREAD

**Issue by dblsaiko** [systemd/systemd#31856]
### systemd version the issue has been seen with

systemd-254.6

### Used distribution

NixOS 23.11

### Linux kernel version used

6.1.80

### CPU architectures issue was seen on

x86_64

### Component

bootctl

### Expected behaviour you didn't see

`bootctl --json=short` or `bootctl status --json=short` prints output in JSON format.

### Unexpected behaviour you saw

`bootctl status --json=short` prints output exactly as without the --json flag.

### Steps to reproduce the problem

Run `bootctl status --json=short`

**Comment by rpigott** [comment]
The --json flag is an argument for `bootctl list`.

**Comment by Unique-Usman** [comment]
@bluca, I was able to locate the bootctl.c file. My question is that, do I need to have the same specifications of machine used by @rpigott  and also if there is any information that could help kindly help with it. Thank you.

**Comment by bluca** [comment]
No need, this can be tested in qemu with mkosi, see docs/HACKING

**Comment by Unique-Usman** [comment]
Thanks you very much @bluca

**Comment by Unique-Usman** [comment]
@bluca, Hello, how are you doing?  thanks for your support and mentorship during the program, please, I am currently having problem using qemu. Any way around that. Thank you very much.

**Comment by Unique-Usman** [comment]
@2xsaiko, the --json option is working for list but not status. 

@bluca I was trying to implement the --json using the DBUS properties but, I got to know through @DaanDeMeyer that JSON should be generated by the daemon which I do not understand how to implement. Thank you.

**Comment by bluca** [comment]
That's for a separate situation, there's no daemon here, so you can implement it in bootctl

**Comment by Unique-Usman** [comment]
@bluca, thank you. I will go for it.

**Comment by 2xsaiko** [comment]
> @2xsaiko, the --json option is working for list but not status.

Right, I needed the Secure Boot status. Is there a better way to get that?

**Comment by Unique-Usman** [comment]
I am currently working on making the --json option to work for bootctl status also.

**Comment by Unique-Usman** [comment]
> > @2xsaiko, the --json option is working for list but not status.
> 
> Right, I needed the Secure Boot status. Is there a better way to get that?

Have you tried using `bootctl status` this show the status of Secure Boot.
